### ISSUE TITLE
RTC loses time after shutdown in dtoverlay state.

### ISSUE THREAD

**Issue by TinManAkshay** [adafruit/Adafruit_CircuitPython_PCF8523#28]
Hi - 

1st method:

I have added dtoverlay=i2c_rtc,pcf8523 in the boot/config.txt
i2cdetect -y 1 shows UU instead 0x68.

I soldered the crystal oscillator and rtc chip started keeping track of time after setting the time, let's say 1pm  Although, when I shutdown the system for 10 mins approx., rtc chip should give me 1:10pm, but its giving me 1:01pm time. There is a battery coin cell as well. It seems like rtc loses 10mins time after complete shutdown of the system and when I turn on the system(via main power supply) the time would start from where it was left before like 1:01pm. 

I dont know how to fix this issue. Please suggest. 

2nd method:
When I comment out dtoverlay and it obviousl shows me 0x68. In this case, I am not losing time even after I turn on the system after 10 mins. Seems like library is writing 0b000 to enable battery switch over. That's exactly what we want. Although, how to get Time Zone?
I do not know how to write to control register3 while device is in reserved address mode (0xUU). Any suggestions would be appreciated.

Thanks,
Akshay

_Originally posted by @TinManAkshay in https://github.com/adafruit/Adafruit_CircuitPython_PCF8523/issues/27#issuecomment-1287391345_

**Comment by caternuson** [comment]
@TinManAkshay This library is a Python/CircuitPython library for use with Adafruit PCF8523 RTC breakouts. Use on a Raspberry Pi would be per this guide:
https://learn.adafruit.com/circuitpython-on-raspberrypi-linux/overview
Open issues in this repo should be related to that use.

You would **not** use this library in conjunction with the dtoverlay. That is a different use case:
https://learn.adafruit.com/adding-a-real-time-clock-to-raspberry-pi/overview
This is not the place to ask for help/support with that use.

Also, there is no support here for troubleshooting a custom RTC module.

Closing since not a library issue. Please do not open a new issue here unless it relates as mentioned above.
